Adyen/UI/Form/Items/Address/FormFullAddressItem.swift

```diff
@@ -54,7 +52,7 @@ public final class FullFormAddressItem: FormValueItem<AddressInfo, AddressStyle>
 
     internal lazy var countrySelecItem: FormRegionPickerItem = {
         let locale = localizationParameters?.locale.map { Locale(identifier: $0) } ?? Locale.current
-        let countries = RegionRepository.localCountryFallback(for: locale as NSLocale)
+        let countries = RegionRepository.localCountryFallback(for: locale as NSLocale).sorted { $0.name < $1.name }
```

Comment:
just `sorted()` will work, right?

Reply:
no, those are just `Region` structs. For `sorted` to work - need to adopt `Comparable` protocol.